src/parser/jobs/gnb/modules/NoMercy.tsx

```diff
@@ -37,228 +34,87 @@ const EXPECTED_USES = {
-	private noMercyWindows: NoMercyState[] = []
+	rotationTableNotesColumnHeader = <Trans id="gnb.nomercy.notes.header"> Bloodfest Used </Trans>
 
-	private get lastNoMercy(): NoMercyState | undefined {
-		return _.last(this.noMercyWindows)
+	expectedGCDs = {
+		expectedPerWindow: EXPECTED_USES.GCD,
+		suggestionContent: <Trans id="gnb.nomercy.suggestions.gcds.content">
+			Try to land 9 GCDs during every <ActionLink {...ACTIONS.NO_MERCY} /> window. A 20 second duration is sufficient
+				to comfortably fit 9 GCDs with full uptime if you wait until the last one-third of your GCD timer to activate it.
+		</Trans>,
+		severityTiers: SEVERITIES.TOO_FEW_GCDS,
 	}
 
-	protected init() {
-		this.addEventHook('cast', {by: 'player'}, this.onCast)
-		this.addEventHook(
-			'removebuff',
+	trackedActions = {
+		icon: ACTIONS.NO_MERCY.icon,
+		actions: [
 			{
-				by: 'player',
-				to: 'player',
-				abilityId: [STATUSES.NO_MERCY.id],
+				action: ACTIONS.GNASHING_FANG,
+				expectedPerWindow: EXPECTED_USES.GNASHING_FANG,
 			},
-			this.onRemoveNoMercy,
-		)
-		this.addEventHook('complete', this.onComplete)
-	}
-
-	private onCast(event: CastEvent) {
-		const actionId = event.ability.guid
 
-		if (actionId === ACTIONS.ATTACK.id) {
-			return
-		}
+			{
+				action: ACTIONS.BURST_STRIKE,
+				expectedPerWindow: EXPECTED_USES.BURST_STRIKE,
+			},
 
-		if (actionId === ACTIONS.NO_MERCY.id) {
-			const noMercyState = new NoMercyState(event.timestamp)
-			const fightTimeRemaining = this.parser.fight.end_time - event.timestamp
-			noMercyState.isRushing = NO_MERCY_BUFF_DURATION >= fightTimeRemaining
+			{
+				action: ACTIONS.SONIC_BREAK,
+				expectedPerWindow: EXPECTED_USES.SONIC_BREAK,
+			},
 
-			this.noMercyWindows.push(noMercyState)
-		}
+			{
+				action: ACTIONS.BLASTING_ZONE,
+				expectedPerWindow: EXPECTED_USES.BLASTING_ZONE,
+			},
 
-		// So long as we're in this window, log our actions to it.
-		const lastNoMercy = this.lastNoMercy
-		if (lastNoMercy != null && lastNoMercy.end == null) {
-			lastNoMercy.rotation.push(event)
+			{
+				action: ACTIONS.BOW_SHOCK,
+				expectedPerWindow: EXPECTED_USES.BOW_SHOCK,
+			},
 
-			const action = getDataBy(ACTIONS, 'id', actionId) as TODO
-			if (!action) { return }
+			{
+				action: ACTIONS.ROUGH_DIVIDE,
+				expectedPerWindow: EXPECTED_USES.ROUGH_DIVIDE,
+			},
+		],
 
-			// Pre-process on the number of certain things we did
-			if (action.onGcd) {
-				lastNoMercy.numGcds++
-			}
+		suggestionContent: <Trans id="gnb.nomercy.suggestions.expected-uses.content">
+			Watch your uses of certain abilities during <ActionLink {...ACTIONS.NO_MERCY} />. Under ideal conditions, you should
+				be using <ActionLink {...ACTIONS.SONIC_BREAK} />, a full <ActionLink {...ACTIONS.GNASHING_FANG} /> combo, and all of
+				your off-GCD skills - <ActionLink {...ACTIONS.BLASTING_ZONE} />, <ActionLink {...ACTIONS.BOW_SHOCK} />, and at least one
+				charge of <ActionLink {...ACTIONS.ROUGH_DIVIDE} /> - under the buff duration.
```

Comment:
Ultra nit, are these hyphens really necessary? First could be a colon and last could just not be there at all.